content/influxdb/cloud/visualize-data/visualization-types/band.md

```diff
@@ -10,4 +10,154 @@ menu:
+  - **Step**: Display a time series in a staircase graph.
+- **Line Colors**: Select a color scheme to use for your graph.
+- **Hover Dimension**: Select the data to display in the tooltip when you hover over the graph:
+  - **auto** or **X-Axis**: Show all points with the same x value along the y-axis.
+  - **Y-Axis**: Show all points with the same y value along the x-axis.
+  - **X-Y Axis**: Show only the point being currently hovered over.
+
+###### X-Axis
+
+- **Generate X-Axis Tick Marks**: Select the method to generate x-axis tick marks:
+  - **Auto**: Select to automatically generate tick marks.
+  - **Custom**: To customize the number of x-axis tick marks, select this option, and then enter the following:
+    - **Total Tick Marks**: Enter the total number of timestamp ticks to display.
+    - **Start Tick Marks At**: Enter the time, in RFC3339 format, to start displaying ticks. Use the **Date Picker** field to automatically generate an RFC3339 formatted timestamp for this field.
+    - **Tick Mark Interval**: Enter the number of milliseconds in between each timestamp tick.
+
+###### Y-Axis
+
+- **Y Axis Label**: Enter the label for the y-axis.
+- **Y-Value Unit Prefix**: Select the prefix to add to the y-value:
+  - **None**: Select to add no prefix.
+  - **SI**: (default) Select to add an International System of Units (SI) or metric prefix.
+  - **Binary**: Select to add a binary multiple prefix.
+- **Y Axis Prefix**: Enter the prefix to add to the y-value.
+- **Y Axis Suffix**: Enter the suffix to add to the y-value.
+- **Generate Y-Axis Tick Marks**: Select the method to generate y-axis tick marks:
+  - **Auto**: Select to automatically generate tick marks.
+  - **Custom**: To customize the number of y-axis tick marks, select this option, and then enter  the following:
+    - **Total Tick Marks**: Enter the total number of ticks to display.
+    - **Start Tick Marks At**: Enter the value to start ticks at.
+    - **Tick Mark Interval**: Enter the interval in between each tick.
+- **Y Axis Domain**: Select the method to generate the y-axis value range:
+  - **Auto**: Select to automatically determine the value range based on values in the data set.
+  - **Custom**: To customize the y-axis domain, manually specify the minimum y-axis value, maximum y-axis value, or range by including both.
+    - **Min**: Enter the minimum y-axis value.
+    - **Max**: Enter the maximum y-axis value.
+
+
+###### Legend
+
+- **Orientation**: Select the orientation of the legend:
+  - **Horizontal**: Select to display the legend horizontally.
+  - **Vertical**: Select to display the legend vertically.
+- **Opacity**: Adjust the legend opacity using the slider.
+- **Colorize Rows**: Select to display legend rows in colors.
+- **Static Legend**: Select **Hide** to hide the legend or if the **Hover Legend Show** option is selected, show the legend only upon hover. Select **Show** to always show the legend.
```

Comment:
Static legend and hover legend has two options, Show and Hide. 
Toggling Show will turn on the feature and show controls like orientation, opacity, colorize row .. etc. Hover Legend and Static legend will have their own attributes rather than sharing them

Reply:
More can be seen in the screenshots above !